src/main/java/core/org/akaza/openclinica/service/StudyBuildServiceImpl.java

```diff
@@ -731,6 +731,8 @@ public String getCurrentBoardUrl(String accessToken, Study study) {
         headers.add("Accept-Charset", "UTF-8");
         HttpEntity<String> entity = new HttpEntity<String>(headers);
         RestTemplate restTemplate = new RestTemplate();
+
+        logger.info("GET " + uri);
```

Comment:
I will merge this but please remember to change this to debug after we find the root cause.